docs/en/credential-data-model.rst

```diff
@@ -113,14 +118,16 @@ The JWT payload contains the following claims. Some of these claims can be discl
       - [NSD]. OPTIONAL. The identifier of the subject of the Digital Credential, the User, MUST be opaque and MUST NOT correspond to any anagraphic data or be derived from the User's anagraphic data via pseudonymization. Additionally, it is required that two different Credentials issued MUST NOT use the same ``sub`` value.
       - `[RFC7519, Section 4.1.2] <https://www.iana.org/go/rfc7519>`_.
     * - **iat**
-      - [SD]. REQUIRED. UNIX Timestamp with the time of JWT issuance, coded as NumericDate as indicated in :rfc:`7519`.
+      - [NSD]. OPTIONAL. UNIX Timestamp with the time of JWT issuance, coded as NumericDate as indicated in :rfc:`7519`.
```

Comment:
@m-basili reported that we do not have any clear statement about the presence/absence of `iat` in both the [PID rulebook](https://github.com/eu-digital-identity-wallet/eudi-doc-attestation-rulebooks-catalog/blob/main/rulebooks/pid/pid-rulebook.md) and the IA.

we have issuance_date for mdoc cbor but nothing for sd-jwt. According to this evidence we could keep iat within the payload.

Reply:
see also the elements below to be considered in this alignment

Reply:
<img width="1111" height="791" alt="image" src="https://github.com/user-attachments/assets/fac7811d-f16f-4e6d-8eda-2e0c254c76b1" />

Reply:
<img width="1141" height="650" alt="image" src="https://github.com/user-attachments/assets/01c3a7cd-0ce4-454b-a021-bacc2c95d513" />